hadoop-hdds/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockInputStreamProxy.java

```diff
@@ -157,11 +158,17 @@ public synchronized int read(ByteBuffer buf) throws IOException {
         lastPosition = blockReader.getPos();
         totalRead += blockReader.read(buf);
       }
+      if (reconstructionReader) {
```

Comment:
I wonder if this makes sense? This read method is going to be called for every read into a file. Eg if something is reading a file 1 byte at a time, it could be called 100's of times. Would it make more sense to increment just once when the proxy class sets up the reconstruction reader? Eg in `createBlockReader()` in the same class?